app.rb

```diff
@@ -0,0 +1,99 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  attr_accessor :people, :books
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def create_person
+    puts 'Do you whant to create a Student (1) 0r Teacher (2)? [input the number]'
+    input = gets.chomp.to_i
+    case input
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      puts 'incorrect choice'
+    end
+  end
+
+  def create_book
+    print 'Book Title: '
+    title = gets.chomp
+    print 'Book Author: '
+    author = gets.chomp
+    book = Book.new(title, author)
+    @books.push(book)
+    puts 'Book created successfully'
+  end
+
+  def create_rental
+    puts 'Select a book from the following list by number'
+    list_books
+    book_index = gets.chomp.to_i
+    puts 'Select a person from the following list by number (not id)'
+    list_people
+    person_index = gets.chomp.to_i
+    print "Date: #{Date.today}"
+    date = gets.chomp
+    rental = Rental.new(date, @books[book_index], @people[person_index])
+    @rentals.push(rental)
```

Comment:
On line 48, kindly inspect the way you arranged your arguments, with how you defined it in the rental.rb file.